tests/setupMockAxios.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios"
```

Comment:
rather than doing this, we could instead do 2 exports in `__mocks__/axios.ts` that mirrors what we've done in our code

```ts
mockAxios.interceptors.request.use(jest.fn())
mockAxios.interceptors.request.use(jest.fn())
```

Reply:
Modified in https://github.com/isomerpages/isomercms-backend/pull/440/commits/204cac45369ab3c81cc7b8b756f9c62aa992e0e2 and changes to tests which use jest-mock-axios reverted in https://github.com/isomerpages/isomercms-backend/pull/440/commits/3ccbc6f28094bb1bddb23c2484b0e11eb90052ee